monitoring/grafana/grr_grafanalib_dashboards/dashboards_to_generate/fleetspeak.dashboard.py

```diff
@@ -0,0 +1,239 @@
+from grafanalib.core import Alert, AlertCondition, Dashboard, Graph, Heatmap, LowerThan, OP_AND, Row, RTYPE_SUM, Target, TimeRange, YAxes, YAxis
+from grr_grafanalib_dashboards.util import add_data_source
+from grr_grafanalib_dashboards.config import ACTIVE_PROCESSES_ALERTING_CONDITION
+
+dashboard = Dashboard(
+  title="Fleetspeak Servers Dashboard",
+  rows=[
+    Row(panels=[
+      Graph(
+        title="Number of Active Processes",
+        targets=[
+          Target(
+            expr='sum(up{job="fleetspeak"})',
+            legendFormat="Active Processes",
+          ),
+        ],
+        alert=Alert(
+          name="Number of Active Processes alert",
+          message="The number of active Fleetspeak Server processes is below {}".format(ACTIVE_PROCESSES_ALERTING_CONDITION),
+          alertConditions=[
+            AlertCondition(
+              Target(
+                expr='sum(up{job="fleetspeak"})',
+                legendFormat="Active Processes",
+              ),
+              timeRange=TimeRange("10s", "now"),
+              evaluator=LowerThan(ACTIVE_PROCESSES_ALERTING_CONDITION),
+              operator=OP_AND,
+              reducerType=RTYPE_SUM
+            )
+          ],
+        )
+      ),
+      Graph(
+        title="Sum of Process Memory Bytes (across all instances)",
```

Comment:
I'd stick to a single convention regarding where we indicate the units used in the graph. I.e. here this is indicated in the graph's title, whereas in "CPU Usage" graph below it's indicated in the legend.
Do I understand it correctly, that we can also explicitly specify the units used in the Y axis? Then we won't need the units description to be present in the title or legend.

Reply:
Yes, Grafana allows editing the Y-axis legend to reflect measurement units. Will work on that.